renovate.json

```diff
@@ -690,7 +691,8 @@
     {
       "groupName": "@reduxjs/toolkit",
       "matchDepNames": [
-        "@reduxjs/toolkit"
+        "@reduxjs/toolkit",
+        "immer"
       ],
       "reviewers": [
         "team:ml-ui",
```

Comment:
@mattkime @legrego I've discussed this with @walterra , who did most of the redux work when he was on ML, and he's suggested that it makes more sense to remove `ml-ui` from the list of owners here. The only usage in ML owned code I believe is some small use in the AIOps code. There is also some usage in the Transforms edit flyout, but this is now owned by Management. @walterra is now on the Visualizations team, and since they are already listed as joint reviewers here, he's suggested removing ml-ui here.

@walterra please update the thread once you've had a chance to discuss with the Visualizations team if it's ok to remove ml-ui from here.

Reply:
@mattkime adding `immer` to this section looks good! we discussed this in the vis team and it's ok to remove `team:ml-ui` from this section, can you please do that as part of this PR?